docs/source/production.rst

```diff
@@ -75,27 +75,38 @@ Where the following **config** files have been used,
 
 NEXT-100
 ------------
-This sample is currently being produced. It is generated using
+A first production using fast simulations (:doc:`detsim`) has been produced using
 
 .. list-table::
    :widths: 40 60
    :header-rows: 0
 
-   * - **nexus**
-     - ``to be added``
-   * - **detsim**
-     - ``to be added``
+   * - **nexus: radiogenics + signal**
+     - ``v7_02_00``
+   * - **nexus: muons**
+     - ``v7_03_00``
+   * - **nexus: LTs and PSFs for Detsim**
+     - ``v7_01_00(+ G4 op bug)``
    * - **IC**
-     - ``to be added``
-   * - **Background Model**
-     - ``to be added``
+     - ``8e1de90c1cec5a1c3d25b35eca456f87f4f7e64c``
```

Comment:
I think this commit was after the productions, since we realized about tthe issue after the production was done. I have checked the production files and the event-id type is Int32, thus the v.1.3.0 was not in use.

Be warned that the commit hash could also be `3e81c90a6c66ae1ff32110423ed8f56ff5c0795f` which updates the activity database to `v3`. Nevertheless the activities used for the analysis are `v4`. I didn't took them from the database but directly from docDB.